taf/git.py

```diff
@@ -290,18 +290,24 @@ def checkout_paths(self, commit, *args):
     def clean(self):
         self._git("clean -fd")
 
-    def clone(self, no_checkout=False, bare=False):
+    def clone(self, no_checkout=False, bare=False, **kwargs):
 
         taf_logger.info("Repo {}: cloning repository", self.name)
         shutil.rmtree(self.path, True)
         self._path.mkdir(exist_ok=True, parents=True)
         if self.repo_urls is None:
             raise Exception("Cannot clone repository. No urls were specified")
-        params = ""
+        params = []
         if bare:
-            params = "--bare"
+            params.append("--bare")
         elif no_checkout:
-            params = "--no-checkout"
+            params.append("--no-checkout")
+
+        for name, value in kwargs.items():
+            params.append(f"--{name} {value}")
```

Comment:
If we have `option=True/False` in `**kwargs`, then I think you should only append a `name` and not a `value`.